src/components/PostInfo/PostInfo.tsx

```diff
@@ -1,5 +1,45 @@
 import React from 'react';
+import './PostInfo.scss';
+import { FullPost } from '../../types/fullPost';
+import { UserInfo } from '../UserInfo';
+import { CommentList } from '../CommentList';
 
-export const PostInfo: React.FC = () => (
-  <>Put the post here</>
-);
+type Post = {
+  post: FullPost;
+};
+
+export const PostInfo: React.FC<Post> = ({ post }) => {
+  const {
+    title,
+    body,
+    user,
+    comments,
+  } = post;
+
+  return (
+    <div className="PostInfo">
+      <div className="PostInfo__header">
+        <h3 className="PostInfo__title">
+          {title}
+        </h3>
+
+        <p>
+          {' Posted by  '}
```

Comment:
```suggestion
        {user && (
          <p>
            {' Posted by  '}
```

Check if user exists here